l10n_it_fatturapa_out_stamp/wizard/wizard_export_fatturapa.py

```diff
@@ -17,13 +17,11 @@ def setDatiGeneraliDocumento(self, invoice, body):
         res = super(WizardExportFatturapa, self).setDatiGeneraliDocumento(
             invoice, body)
         if invoice.tax_stamp:
-            if not invoice.company_id.tax_stamp_product_id:
-                raise UserError(_(
-                    "Tax Stamp Product not set for company %s"
-                ) % invoice.company_id.name)
-            stamp_price = invoice.company_id.tax_stamp_product_id.list_price
             body.DatiGenerali.DatiGeneraliDocumento.DatiBollo = DatiBolloType(
-                BolloVirtuale="SI",
-                ImportoBollo='%.2f' % float_round(stamp_price, 2),
-            )
+                BolloVirtuale="SI")
+            if invoice.company_id.tax_stamp_product_id:
+                stamp_price = invoice.company_id.tax_stamp_product_id.list_price
+                if stamp_price != 0:
```

Comment:
Essendo un prezzo, meglio controllare se è `0` con il metodo apposito `float_is_zero` che evita problemi di precisione/rounding
```suggestion
                precision, rounding = self.env['product.template']._fields['list_price'].digits
                if not float_is_zero(stamp_price, precision, rounding):
```

Reply:
Ok, però essendo una valore in euro (i decimali non possono essere più di 2) farei come [qui](https://github.com/OCA/l10n-italy/blob/4a61716784dbf60320341f283c0ea9d2e9151b99/l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py#L1422)

Reply:
Non è il massimo scolpire il `2` nel codice (anche se non cambierà mai), ma è comunque un passo avanti.. Per me ok

Reply:
Se volessimo prevedere un uso generico (in paesi con diversi decimali sulla valuta) dovremmo basarci sui decimali della valuta, intanto vado cosi